yb-voyager/cmd/exportData.go

```diff
@@ -211,6 +211,13 @@ func exportData() bool {
 					utils.ErrExit("failed to delete stream id after data export: %v", err)
 				}
 			}
+			if exporterRole == SOURCE_DB_EXPORTER_ROLE {
```

Comment:
This comment is for line 204: the `markCutoverProcessed()` should be called only after cleaning the YB streaming ID and pg replication slot. If voyager exits *after* marking cutover processed and *before* deleting the replication objects, they will be left uncleaned.